03.bowling/y-bowling.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+scores = ARGV[0].split(',')
+shots = []
+
+scores.each do |s|
+  shots << (s == 'X' ? 10 : s.to_i)
+  shots << 0 if shots.length < 18 && s == 'X'
+end
+
+frames = shots.each_slice(2).to_a
+
+frames[9] += frames.pop if frames.length == 11
+point = frames.each_with_index.sum do |frame, i|
```

Comment:
```suggestion
frames = shots.each_slice(2).to_a
frames[9] += frames.pop if frames.length == 11

point = frames.each_with_index.sum do |frame, i|
```

こんなふうに改行した方が処理のグルーピングとしてわかりやすいように思いました